src/components/MoneyReportHeader.js

```diff
@@ -52,13 +55,22 @@ const defaultProps = {
 };
 
 function MoneyReportHeader(props) {
+    const {translate} = useLocalize();
     const moneyRequestReport = props.report;
+    const policy = props.policies[`${ONYXKEYS.COLLECTION.POLICY}${moneyRequestReport.policyID}`];
+    const isPolicyAdmin = policy.role === CONST.POLICY.ROLE.ADMIN;
+    const isManager = ReportUtils.isMoneyRequestReport(moneyRequestReport) && lodashGet(props.session, 'accountID', null) === moneyRequestReport.managerID;
+    const isReportApproved = ReportUtils.isReportApproved(moneyRequestReport);
+    const isPayer = CONST.POLICY.TYPE.CORPORATE ? isPolicyAdmin && isReportApproved : isPolicyAdmin || (ReportUtils.isMoneyRequestReport(moneyRequestReport) && isManager);
```

Comment:
Maybe just for better readability
```suggestion
    const isPayer = CONST.POLICY.TYPE.CORPORATE ? (isPolicyAdmin && isReportApproved) : isPolicyAdmin || (ReportUtils.isMoneyRequestReport(moneyRequestReport) && isManager);
```